src/mpid/ch3/src/ch3u_rma_pkthandler.c

```diff
@@ -191,29 +191,17 @@ static int MPIDI_CH3_ExtPktHandler_Accumulate(MPIDI_CH3_Pkt_flags_t flags,
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_CH3_EXTPKTHANDLER_ACCUMULATE);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_CH3_EXTPKTHANDLER_ACCUMULATE);
```

Comment:
The change in `ch3u_rma_pkthandler.c` is not clear to me as it mixes two different topics : (1) dataloop->flatten datatype, (2) simplify extended header types. I will review the file again once the commits are cleaned up.

Reply:
Not fixing based on the discussion below.